papis/commands/browse.py

```diff
@@ -135,11 +136,11 @@ def run(document: papis.document.Document,
 @papis.cli.query_argument()
 @papis.cli.sort_option()
 @click.option("-k", "--key", default="",
-              help="Use the value of the document's key to open in"
-                   " the browser, e.g. doi, url, doc_url ...")
+              help="Use this key in the document as an URL to open in"
+                   " the browser (e.g. doi, url, doc_url)")
```

Comment:
```
              help="Use this key as the URL to open in the browser"
                   " (e.g. doi, url, doc_url)")
```
It's a bit convoluted, so maybe this is enough?

Reply:
Yep, that sounds nicer: changed in [46271e3](https://github.com/papis/papis/pull/1010/commits/46271e343daad20edabb41e03259dc6b9362fcb7).